docs/data_types/image/image.md

```diff
@@ -0,0 +1,132 @@
+# Image
+
+!!! note
+    This requires `Pillow` dependency. You can install all necessary dependencies via
+    ```python 
+    pip install "docarray[image]"
+    ```
+
+DocArray offers several Image specific types to represent your image data from [`ImageUrl`][docarray.typing.url.ImageUrl] to [`ImageBytes`][docarray.typing.bytes.ImageBytes] to [`ImageNdArray`][docarray.typing.tensor.image.image_ndarray.ImageNdArray].
+
+<figure markdown>
+  ![](apple.png){ width="220" }
+</figure>
+
+## Load data
+You can load image data by specifying your local or remote url as an [`ImageUrl`][docarray.typing.url.ImageUrl] 
+and then call [`.load()`][docarray.typing.url.ImageUrl.load] on it. By default, [`ImageUrl.load()`][docarray.typing.url.ImageUrl.load] returns an instance of [`ImageNdArray`][docarray.typing.tensor.image.image_ndarray.ImageNdArray].
+
+```python hl_lines="11"
+from docarray.typing import ImageTensor, ImageUrl
+from docarray import BaseDoc
+
+
+class MyImage(BaseDoc):
+    url: ImageUrl
+    tensor: ImageTensor = None
```

Comment:
lets split this code block in two. 

First block: lets create a document where we are going ot use `ImageUrl`